チャット画面のビュー作成

drumnaoto opened the pull request:
# WHAT
Chatspace
チャット画面のビュー作成。

# WHY
必須機能のため。
![2017-11-01 18 50 31](https://user-images.githubusercontent.com/33175652/32271508-8dce91ae-bf3d-11e7-8787-f9ce39996507.png)

drumnaoto committed:
no-more-unnecessary-files

drumnaoto committed:
make-messages_controller

drumnaoto committed:
send-index-to-root

drumnaoto committed:
delete-css_file-for-scss

drumnaoto committed:
add-gem-font_awesome_rails

drumnaoto committed:
make-reset_css

drumnaoto committed:
make-index_html_haml

drumnaoto committed:
make-application_scss

drumnaoto committed:
make-variable_css

drumnaoto committed:
make-clearfix_scss

drumnaoto committed:
make_message_scss_for_render

TOMARUMARU commented on app/assets/stylesheets/application.scss:
```diff
@@ -0,0 +1,171 @@
+@import "clearfix";
+@import "font-awesome";
+
+
```
application.scssはファイルを読み込むだけにしましょう！

TOMARUMARU commented on app/controllers/messages_controller.rb:
```diff
@@ -0,0 +1,6 @@
+class MessagesController < ApplicationController
+end
```
endの位置を修正してください！

TOMARUMARU reviewed (commented):
確認のためビューのスクリーンショットを貼ってください！

drumnaoto committed:
change-end-position

endの位置を修正しました。

drumnaoto committed:
add-semicolon

最後にセミコロンがなかったので、
今後に向けて追記しました。

drumnaoto committed:
change-scss

application.scssファイルに記述していたものを、messages.scssファイルに移し、application.scssでそのフ
ァイルをimportしました。
